ee/vellum_ee/workflows/tests/test_serialize_module.py

```diff
@@ -96,6 +96,53 @@ def test_serialize_module_includes_additional_files():
     assert "CONSTANT_VALUE" in additional_files["utils/constants.py"]
 
 
+def test_serialize_module_with_pydantic_array():
+    """
+    Test that serialize_module correctly serializes arrays of Pydantic models in workflow inputs.
+
+    This test verifies that when a workflow has inputs containing a List[PydanticModel],
+    the serialization properly converts the Pydantic models to JSON format.
+    """
+    module_path = "tests.workflows.pydantic_array_serialization"
+
+    # WHEN we serialize it
+    result = BaseWorkflowDisplay.serialize_module(module_path)
+
+    assert hasattr(result, "exec_config")
+    assert hasattr(result, "errors")
+    assert isinstance(result.exec_config, dict)
+    assert isinstance(result.errors, list)
+
+    input_variables = result.exec_config["input_variables"]
+    assert len(input_variables) == 1
+
+    items_input = input_variables[0]
+    assert items_input["key"] == "items"
+    assert items_input["type"] == "JSON"
```

Comment:
add a comment here saying that in the future, we will need to update this to be a custom type based on an openapi schema